parts/chapters/subsections/10.3/AQUFETP.fodt

```diff
@@ -4470,7 +4470,7 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
        <text:p text:style-name="P9889">AQUID</text:p>
       </table:table-cell>
       <table:table-cell table:style-name="Table903.C2" table:number-columns-spanned="3" office:value-type="string">
-       <text:p text:style-name="_40_Table_20_Contents">A positive integer greater than or equal to one and less than or equal to NANAQU on the AQUDIMS keyword in the RUNSPEC section, that defines the Fetkovich aquifer number.</text:p>
+       <text:p text:style-name="_40_Table_20_Contents"><text:span text:style-name="T5280">A positive integer greater than or equal to one and less than or equal to NANAQ on the AQUDIMS keyword in the RUNSPEC section, that</text:span> defines the <text:span text:style-name="T5703">Fetkovich aquifer number</text:span><text:span text:style-name="T8306">.</text:span></text:p>
```

Comment:
```suggestion
       <text:p text:style-name="_40_Table_20_Contents">A positive integer greater than or equal to one and less than or equal to NANAQ on the AQUDIMS keyword in the RUNSPEC section, that defines the Fetkovich aquifer number.</text:p>
```
@hakonhagland This is the first missed one.

Reply:
So this file was not changed by the script even if it should have been? I will check closer what happened here

Reply:
Notice that the span tag on line 4473 above has style name T5280 `<text:span text:style-name="T5280">`. But there is no styles with that name defined in the file. So the script will not remove it unless it is a defined style with a given revision id

Reply:
If someone has unselected 
```Options>LibreOffice Writer>Comparison>Random number...>Store it when changing the document```
then will they create span tags without revision ids that are ignored by the script?

Reply:
@gdfldm Thanks for the info! So we should uncheck this box to avoid having libreoffice store the span tags in the documents, right? Like this:

![image](https://github.com/OPM/opm-reference-manual/assets/6708379/e3db73bb-cbb0-4d81-8c4b-408621ba1034)

Reply:
I am not sure I understand what it does but I created a simple fodt document with the option set to false, and was able to insert new lines at the start without creating new styles.
When I set the option to true and inserted a new line at the start it created a new style:
```
  <style:style style:name="P1" style:family="paragraph" style:parent-style-name="Standard">
   <style:text-properties officeooo:rsid="001b60e6" officeooo:paragraph-rsid="001b60e6"/>
  </style:style>
```

Reply:
Thanks for this confirmation! Then it looks like we should keep this box unchecked from now on? I can update the README.md with this information?

Reply:
I think so (with luck this will help).

Reply:
> Notice that the span tag on line 4473 above has style name T5280 `<text:span text:style-name="T5280">`. But there is no styles with that name defined in the file. So the script will not remove it unless it is a defined style with a given revision id

I see. I guess that is the root problem. The spans were removed previously and the later the unsused styles where removed when someone touched the file again. Or maybe the styles are in another document?

I can I hack the script to remove those spans or should I do that manually?

Reply:
@blattms I can create a script to remove unused span tags? It can be useful in the future also maybe?

Reply:
> @blattms I can create a script to remove unused span tags? It can be useful in the future also maybe?

I assume that you are faster with making the script than me removing the quirks manually (there are quite a few)? Then please go for it.